### ISSUE TITLE
dotnet new blazorserver -o BlazorApp --no-https Commmand Not Working

### ISSUE THREAD

**Issue by Coompt-King** [dotnet/core#6528]
Problem encountered on https://dotnet.microsoft.com/learn/aspnet/blazor-tutorial/create
Operating System: linux
Distribution: Ubuntu

So, I started the tutorial from the beginning, but when I ran the command it didn't work. Instead I got: Segmentation fault (core dumped)

**Comment by mairaw** [comment]
Did you use snap manager to install the SDK?
Another post mentioned they tried an alternative method when they got this error:
https://github.com/dotnet/core/issues/6279#issuecomment-844515735

@leecow @rbhanda any other suggestions?

**Comment by mairaw** [comment]
@kouvel @dagood is this something you can help with?

**Comment by dagood** [comment]
> when I tried to do the `sudo snap alias dotnet-runtime-50.dotnet dotnet` command I got this `error: cannot perform the following tasks: - Setup manual alias "dotnet" => "dotnet" for snap "dotnet-runtime-50" (cannot enable alias "dotnet" for "dotnet-runtime-50", already enabled for "dotnet-sdk")`

Maybe I'm missing some possibility, but it sounds like you already had the `dotnet-sdk` snap installed and had its alias set up, then also tried to install the `dotnet-runtime-50` snap and set up its alias, too. You should only need the SDK snap. (This section of the docs is about the difference between the SDK and Runtime snaps: <https://docs.microsoft.com/en-us/dotnet/core/install/linux-snap#sdk-or-runtime>.)

That said, I agree that doesn't seem like it would cause this problem, anyway. 😄

> Segmentation fault (core dumped)

This has also reported at https://github.com/dotnet/runtime/issues/51322.

/cc @dleeapho @NikolaMilosavljevic @MichaelSimons (dotnet/runtime area-Snap owners)

**Comment by safern** [comment]
Yes this is definitely a dupe of: dotnet/runtime#51322. I still have to take a look at why we are crashing when initializing globalization.

**Comment by Coompt-King** [comment]
Alright I'll try that

**Comment by Coompt-King** [comment]
So, I tried uninstalling the runtime and the sdk, then reinstalling the sdk, went through and ran every solution I could fine on the site (there were like 2), but it still gives me the same error. I also went to the other issue that was mentioned, but it didn't help

**Comment by dagood** [comment]
I didn't see any obvious workaround that would fix the `dotnet new` error with the snap when I poked through https://github.com/dotnet/runtime/issues/51322 and related issues. I'm not sure there's a way to get the snap version of .NET to work. (I didn't mean to suggest that you should try installing just the SDK snap, sorry for the wasted effort there. 🙁 I was trying to help clarify why you got an error while running `snap alias` and mention that I don't think that's related to the segmentation fault.)

Have you tried uninstalling the snap and then installing using one of the non-snap methods, like the apt package feed?
https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu

The link @mairaw posted to https://github.com/dotnet/core/issues/6279#issuecomment-844515735 shows someone else who hit this issue with the snap and was able to install a non-snap .NET to get things working.

**Comment by dagood** [comment]
Glad it helped! Thanks for letting us know.
